### PR TITLE
Improve deployment info

### PR DESCRIPTION
**Pull request by keybits**

### CODE REVIEW

File: jekyll/_cci2/deployment_integrations.md [hunk 1]
```diff
@@ -6,8 +6,33 @@ categories: [deploying]
...
+
+**Note:** Currently (during Beta), the CircleCI 'Project Settings' UI has a section titled 'Continuous Deployment':
+
+![Deployment Integrations]({{ site.baseurl }}/assets/img/docs/deployment-integrations.png)
+
+### AWS Deployment
+
+1. To deploy to AWS from CircleCI 2.0 use the [awscli installation instructions](http://docs.aws.amazon.com/cli/latest/userguide/installing.html) to ensure that `awscli` is installed in your primary container. 
+
+2. Add your AWS credentials to the **Project Settings > AWS Permissions** page in the CircleCI application.
```
Comment (eric-hu) [inline jekyll/_cci2/deployment_integrations.md]: Does this section currently work?

Comment (keybits) [inline jekyll/_cci2/deployment_integrations.md]: That info was already in the 2.0 docs so I presume it is correct and had been tested. Will double check.

File: jekyll/_cci2/deployment_integrations.md [hunk 2]
```diff
@@ -6,8 +6,33 @@ categories: [deploying]
...
 ---
 
+## Available Deployment Targets
 
-## Conditional Deployment 
+You can deploy to any service you choose with CircleCI 2.0. By adding commands to `.circleci/config.yml` and setting secrets on the 'Project Settings > Environment Variables' screen, you have a lot of flexibility. For deployment targets other than those listed in 'Deployment Integrations' below, please see the '<a href="#other-deployments">Other Deployments</a>' section.
+
+## Deployment Integrations
+
+**Note:** Currently (during Beta), the CircleCI 'Project Settings' UI has a section titled 'Continuous Deployment':
```
Comment (ndintenfass) [inline jekyll/_cci2/deployment_integrations.md]: Given that "Beta" is basically over, perhaps we should just go ahead and point out this is legacy?

Comment (keybits) [inline jekyll/_cci2/deployment_integrations.md]: On 'GA' day, we'll do a sweep of this and remove all Beta references.

**Comment by keybits** [comment]
Thanks - agree that more iteration is needed :-) Merging for now.

**Pull request merged**